tests/dummy/app/templates/docs/transitions.md

```diff
@@ -0,0 +1,49 @@
+# Transitions
+
+Transitions assign each sprite to a motion. A transition tells sprite categories how to animate from one state to another. For example, a transition can assign the `fade` motion to all `insertedSprites`. Each time a sprite gets added to a list, it would fade into the list as that sprite went from its initial to its final state. 
+
+Transitions are interruptible and they run every time the state of a sprite changes. Therefore, they must be implemented as generator functions. Also, this means that the state of a sprite can change during a transition.
+
+`fade` and `moveOver` are built-in transitions. To use the built in transitions, import `fade` or `moveOver` and then pass them into your template.
+
+In this demonstration, the message fades in or it fades out as it changes states. Each click of the button makes the state of the message change, causing the transition to run.
+
+### Fade
```

Comment:
Sometime before this point, I think it might be worth explaining that creating an animation usually requires doing some work in two files, a component's hbs and js. Animations are one of the first "fun" things that a front end dev usually wants to try, and so they might copy/paste this hbs in and expect it to work, and not explore what's in the js.